lib/t_web/channels/profile_channel.ex

```diff
@@ -107,12 +107,15 @@ defmodule TWeb.ProfileChannel do
   end
 
   defp replace_photo_url_with_s3key(%{"s3_key" => _} = background), do: background
-  defp replace_photo_url_with_s3key(%{"color" => _} = bg), do: bg
```

Comment:
the issue was this matching the story background with "proxy" + position (incl. "color") -> not getting s3_key